sensor_msgs/msg/NavSatFix.msg

```diff
@@ -42,4 +42,4 @@ uint8 COVARIANCE_TYPE_APPROXIMATED = 1
 uint8 COVARIANCE_TYPE_DIAGONAL_KNOWN = 2
 uint8 COVARIANCE_TYPE_KNOWN = 3
 
-uint8 position_covariance_type
+uint8 position_covariance_type = COVARIANCE_TYPE_UNKNOWN
```

Comment:
And for default value syntax you don't use the `=` symbol. And I don't think that you can use the constants in the default value, though I'm not sure on that. 

But in any case this is the default value already so we shouldn't over declare it.

Reply:
I was hoping to be explicit about it being initialized as the UNKNOWN states. 

It did compile fine with the equal, but you are right, it's not a default value, it turned `position_covariance_type` into a constant. 

If I simply remove the `=` sign, it errors out. The current IDL compiler does not appear to support using constants as default values. 
```
  Error processing 'uint8 position_covariance_type COVARIANCE_TYPE_UNKNOWN'
  of 'sensor_msgs/NavSatFix': 'value 'COVARIANCE_TYPE_UNKNOWN' can not be
  converted to type 'uint8': must be a valid integer value >= 0 and <= 255'
```

Fixed in e8b50ab480461ec4fb9e92d38dc43eb566fd345f